template.rb

```diff
@@ -1,9 +1,9 @@
-# $ rails _5.2.2_ new app-name -m https://raw.githubusercontent.com/geoblacklight/geoblacklight_sidecar_images/master/template.rb
+# $ rails _7.0.4.2_ new app-name -m https://raw.githubusercontent.com/geoblacklight/geoblacklight_sidecar_images/master/template.rb
 
-gem "blacklight", ">= 7.0"
-gem "geoblacklight", ">= 2.0"
+gem "blacklight", ">= 7.0", "< 8.0"
+gem "geoblacklight", ">= 4.0"
 gem "statesman", ">= 3.4"
-gem "geoblacklight_sidecar_images", git: "git://github.com/ewlarson/geoblacklight_sidecar_images.git", branch: "master"
+gem "geoblacklight_sidecar_images", git: "git://github.com/ewlarson/geoblacklight_sidecar_images.git", branch: "develop"
```

Comment:
question: should this be `master`?

Reply:
I changed that to develop, as this repo is using the git-flow branching strategy:
http://danielkummer.github.io/git-flow-cheatsheet/

The develop branch is the working branch and master is a just a copy of any code that gets released. The develop branch would have any unreleased enhancements or bug fixes, so it just feels like a better branch IMO.